client/style/style.css

```diff
@@ -290,30 +448,50 @@ a {
     background-color: transparent;
 }
 
-#product-purchase-buyer {
+.product-purchase-buyer {
+    margin-bottom: 1.25rem;
+}
+
+.product-purchase-availability {
     margin-bottom: 1.25rem;
 }
 
-#product-purchase-availability {
+.bottom-space {
     margin-bottom: 1.25rem;
 }
 
-#availability-delivery {
+.availability-delivery {
+    display: flex;
+    align-items: flex-start;
+    margin-top: 1.25rem;
+    padding-bottom: 1.25rem;
     border-bottom: 1px solid #dfdfdf;
 }
 
-#availability-stock {
+.availability-stock {
+    display: flex;
+    align-items: flex-start;
+    margin: 1.25rem 0;
+    padding-bottom: 1.25rem;
     border-bottom: 1px solid #dfdfdf;
 }
 
-/* incomplete */
-#availability-assembly {
-    display: block;
+.availability-assembly {
+    display: flex;
+    align-items: flex-start;
+    margin-top: 1.25rem;
+    padding-bottom: 1.25rem;
 }
 
-/* incomplete */
-.stylepicker-variant-item {
-    display: block;
+.availability-svg {
+    display: inline-block;
+    height: 1.5rem;
+    width: 1.5rem;
+    vertical-align: middle;
+}
+
+.availability-spacer {
+    padding: 0 .5rem 0 1rem;
```

Comment:
At some point around 500 lines of CSS, it might be worth looking into something like SCSS to condense rules a bit.